packages/playground/windows/playground/MainPage.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+#include "MainPage.g.h"
+
+namespace winrt::playground::implementation {
+
+struct MainPage : MainPageT<MainPage> {
+  MainPage();
+
+  hstring Test();
+  void Test(hstring const &value);
+
```

Comment:
what does this represent?

Reply:
ah.. would you believe it was a local test of some cppwinrt stuff.. I'll remove it.